crates/graphql/src/schema/objects/nft.rs

```diff
@@ -589,14 +589,14 @@ impl Collection {
             )
             .filter(metadata_collection_keys::collection_address.eq(self.0.mint_address.clone()))
             .filter(metadatas::burned_at.is_null())
-            .filter(metadata_collection_keys::verified.eq(true))
             .select(current_metadata_owners::owner_address)
-            .distinct()
-            .count()
-            .get_result::<i64>(&conn)
-            .context("Failed to load collection holder count")?
-            .try_into()
+            .distinct_on(current_metadata_owners::owner_address)
+            .load::<String>(&conn)
+            .context("Failed to load collection holder count")?;
+
+        u64::try_from(unique_holders.len())
```

Comment:
Pretty rough though returning all these records instead of just the number from the db. Better to do with sql string then?

What is wrong with group_by with diesel @ray-kast